internal/pkg/certs/cert.go

```diff
@@ -30,7 +28,7 @@ func GenerateCert(ca []byte,
 
 	// key size must be higher than 1024, otherwise the PolicyServer
 	// TLS acceptor will refuse to start
-	servingPrivateKey, err := rsa.GenerateKey(rand.Reader, 4096)
+	privateKey, err := rsa.GenerateKey(rand.Reader, 4096)
```

Comment:
Maybe we could create a constant defining the size of the key? This would be used to create the internal CA and the Policy Server certs.